apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/exception/StatusRuntimeExceptionMapper.java

```diff
@@ -88,15 +91,34 @@ public RestResponse<ApiError> statusRuntimeException(StatusRuntimeException grpc
         break;
     }
 
-    return grpcResponse(
+    // Presumed to be server-side issue so log:
+    return grpcResponseAndLogAsError(
         sc, Response.Status.INTERNAL_SERVER_ERROR, "Unhandled gRPC failure", grpcE.getMessage());
   }
 
-  private static RestResponse<ApiError> grpcResponse(
+  private static RestResponse<ApiError> grpcResponseAndLogAsError(
       Status.Code grpcCode, Response.Status httpStatus, String prefix, String suffix) {
-    final String msg = String.format("%s (%s->%s): %s", prefix, grpcCode, httpStatus, suffix);
+    final String message = buildFailureMessage(grpcCode, httpStatus, prefix, suffix);
+    LOGGER.error("Bridge gRPC call failed, problem: {}", message);
+    return buildGrpcResponse(grpcCode, httpStatus, message);
+  }
+
+  private static RestResponse<ApiError> grpcResponseNoLog(
+      Status.Code grpcCode, Response.Status httpStatus, String prefix, String suffix) {
+    return buildGrpcResponse(
+        grpcCode, httpStatus, buildFailureMessage(grpcCode, httpStatus, prefix, suffix));
+  }
+
+  private static String buildFailureMessage(
+      Status.Code grpcCode, Response.Status httpStatus, String prefix, String suffix) {
+    return String.format(
+        "%s (Status.Code.%s->%d): %s", prefix, grpcCode, httpStatus.getStatusCode(), suffix);
+  }
+
+  private static RestResponse<ApiError> buildGrpcResponse(
```

Comment:
should be `buildRestResponse`?

Reply:
Done in https://github.com/stargate/stargate/pull/2171/commits/eb812454b4c804ffb27004f518b852ea9e01ed5b